lib/takosan.rb

```diff
@@ -9,8 +9,8 @@ module Takosan
 
   module_function
 
-  def privmsg(message)
-    request('/privmsg', { 'channel' => @@channel, 'message' => message, 'name' => @@name, 'icon' => @@icon })
+  def privmsg(message, **attachments)
```

Comment:
I intended to use it as Slack's [message attachments](https://api.slack.com/docs/message-attachments). However, it seems to confusing, so I change it to `options`.